src/main/resources/data/acls/05/acl/auth_open.ttl

```diff
@@ -0,0 +1,8 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix eg: <http://example.com/terms#> .
+
+<> a acl:Authorization ;
+   acl:agent foaf:Agent ;
+   acl:mode acl:Read ;
+   acl:accessToClass eg:publicImage .
```

Comment:
I would expect one of the protected resources to be of type `eg:publicImage`. no?